systemd/oracle-core.service

```
@@ -0,0 +1,16 @@
+[Unit]
+Description=Oracle Core v2.0
+Documentation=https://github.com/ergoplatform/oracle-core
+After=network.target
+Wants=network.target
+
+[Service]
+Type=simple
+# Config file is in ~/.config/oracle-core/
+WorkingDirectory=/home/USER/.config/oracle-core
```

```suggestion
WorkingDirectory=[PATH TO oracle_config.yml]
```
This should capture the operator's attention to set the proper path.